packages/tool_util/setup.py

```diff
@@ -97,7 +97,7 @@ def get_var(var_name):
     include_package_data=True,
     install_requires=requirements,
     extras_require={
-        'condatesting': ['jinja2'],
+        'mulled': ['conda', 'cytoolz', 'jinja2', 'Whoosh'],
```

Comment:
What's cytoolz for? I couldn't find a reference in the repo (on the phone).

Reply:
cytoolz is an undeclared dependency of the conda package on pypi.